### DISCUSSION TITLE
Welcome to JellyFlix Discussions!

### DISCUSSION THREAD

**Discussion by prayag17** [prayag17/JellyFlix#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tw1zT3d2four7** [comment]
Cant scroll in the tv guide with your plugin other then that great work
adding this fixes it

.programContainer.emby-scroller {
    overflow: auto!important;
}

**Reply by ctrlalt-666** [reply]
are you still using Jellyflix? Could you drop your code? When I use the original one I can't access my settings in emby

**Comment by prayag17** [comment]
Don't you this css right now. I am not working on it (currently) and it was made for JF 10.7
